eng/restore/docs.targets

```
@@ -1,7 +1,7 @@
 <Project>
 
   <ItemGroup>
-    <PackageDownload Include="$(MicrosoftPrivateIntellisensePackage)" Version="[$(MicrosoftPrivateIntellisenseVersion)]" />
+    <PackageDownload Include="microsoft.private.intellisense" Version="[$(MicrosoftPrivateIntellisenseVersion)]" />
```

```suggestion
    <PackageDownload Include="Microsoft.Private.Intellisense" Version="[$(MicrosoftPrivateIntellisenseVersion)]" />
```
This is the proper casing of this package name.